src/alaveteli.yaml

```diff
@@ -0,0 +1,36 @@
+---
+description: >
+  Platform for making public freedom of information requests to public bodies.
+  Alaveteli helps you lower the barriers that prevent citizens asking questions
+  of those in power.
+license: AGPL 3.0
+name: Alaveteli
+programmingLanguages:
+  - ruby
+rating: 5
+repoUrl: "https://github.com/mysociety/alaveteli"
+sdgs: [16]
+websiteUrl: "https://alaveteli.org/"
+
+contributionGuidelinesUrl: "https://alaveteli.org/docs/developers/"
+# They have no logo. We could otherwise use the mySociety logo
+logoUrl: "https://alaveteli.org/assets/img/favicon.png"
+
+otherLinks:
```

Comment:
Might I suggest to also add https://www.asktheeu.org to the list?
Since the EU is bigger than the UK, it probably deserves an entry, too.

Proof that it uses Alaveteli: https://www.asktheeu.org/help/alaveteli